src/components/icon/demoFontIconsWithLigatures/script.js

```diff
@@ -0,0 +1,23 @@
+
+angular
+  .module('appDemoFontIconsWithLigatures', ['ngMaterial'])
+  .controller('DemoCtrl', function( $scope ) {
+      // Specify a list of font-icons with ligatures and color overrides
+      var iconData = [
+            {name: 'accessibility'  , color: "#777" },
+            {name: 'question_answer', color: "rgb(89, 226, 168)" },
+            {name: 'backup'         , color: "#A00" },
+            {name: 'email'          , color: "#00A" }
```

Comment:
Can you consistently use either single-quotes or double-quotes? (I personally prefer single).